R/get_lookup_paths.R

```diff
@@ -130,9 +130,8 @@ get_datazone_pop_path <- function(file_name = NULL, ext = "rds") {
 #' @export
 #'
 #' @family lookup file paths
-get_gpprac_ref_path <- function(ext = "sav") {
-  gpprac_dir <-
-    fs::path(get_lookups_dir(), "National Reference Files")
+get_gpprac_ref_path <- function(ext = "csv") {
+  gpprac_dir <- fs::path(get_lookups_dir(), "National Reference Files")
```

Comment:
This looks good. My only query - in the email this said there was a `csv files` folder, should we reference this here? I had a look and there are csv files outside of the `csv files` folder and also inside of this.